openwisp_controller/config/tests/test_config.py

```diff
@@ -652,19 +652,35 @@ def test_remove_duplicate_files(self):
                 ]
             },
         )
-        config = self._create_config(organization=self._get_org())
-        config.templates.add(template1)
-        config.templates.add(template2)
-        config.refresh_from_db()
-        try:
+        org = self._get_org()
+        with self.subTest('Test template applied on creating config'):
+            config = self._create_config(
+                organization=org,
+                templates=[template1, template2],
+            )
+            config.full_clean()
+            config.save()
             result = config.get_backend_instance(
                 template_instances=[template1, template2]
             ).render()
-        except ValidationError:
```

Comment:
This is code would cleanly show the error if the test fails for any reason and was introduced on purpose, so why remove t?

Reply:
It is not removed. The original test is shown below. This is the new test that I added.